### PR TITLE
Sending sms

### LINKED ISSUE #5: 4.1.5 Sending SMS
Feature 4.1.5
1. Success scenario
2. SMS sent to unknown recipient scenario
3. Failure: abort scenario

### PR DESCRIPTION
**Pull request by robertKozik**
I created this PR as draft because I excluded for now tests from it. Ill add them as soon as I would finish them, but I'm not sure that I will be ready with them for your weekly meeting. Thats why I decided to send you changes to review, but without tests.

### CODE REVIEW

File: UE/Application/Ports/BtsPort.cpp [hunk 1]
```diff
@@ -56,6 +57,10 @@ void BtsPort::handleMessage(BinaryMessage msg)
                 handler->handleAttachReject();
             break;
         }
+        case common::MessageId::UnknownRecipient:
```
Comment (jakjare) [inline UE/Application/Ports/BtsPort.cpp:60]: Conflict with `main`.

File: UE/Application/Ports/BtsPort.cpp [hunk 2]
```diff
@@ -80,4 +85,12 @@ void BtsPort::sendAttachRequest(common::BtsId btsId)
 
 }
 
+void BtsPort::handleMessageSend(Sms &sms)
+{
+    //logger.logInfo("sms send from:"+sms.from+" to:"+sms.to);
```
Comment (jakjare) [inline UE/Application/Ports/BtsPort.cpp:90]: Why is that commented?
```suggestion
    logger.logInfo("sms send from:" + sms.from + " to:" + sms.to);
```

**Review by jakjare — CHANGES_REQUESTED** [review]
You should resolve conflicts with `main` first. I and @Koci0 are using that case in #7 and #6.

**Comment by robertKozik** [comment]
Oh, I get it, so ill change my implementation of unknownRecepient with it in mind.

**Pull request closed without merging**